### PR TITLE
Adding the spatial regression model examples with Hammonds Flycatcher point counts and max logits

### PR DESCRIPTION
**Pull request by laurenharrell**

### CODE REVIEW

**Comment by ddkapan** [comment]
Thanks @laurenharrell !!!
